packages/date-time-picker/stories/date-time-picker.stories.ts

```diff
@@ -215,7 +125,10 @@ export const timeGranularity = (args: StoryArgs = {}): TemplateResult => {
         timeGranularity: args.timeGranularity,
     };
 
-    return renderTimeField(`Time Granularity: ${args.timeGranularity}`, args);
+    return renderDateTimePicker(
+        `Time Granularity: ${args.timeGranularity}`,
```

Comment:
This story doesn't seem to change based off of the select `timeGraularity` value.